tests/test_webui_config_auth.py

```
@@ -0,0 +1,121 @@
+# extract_credentials — config file
+# ----------------------------------------------------------------------
+
+
+def test_extract_returns_facebook_creds_when_both_set(tmp_path: Path):
+    p = _write(
+        tmp_path,
+        '[marketplace.facebook]\nusername = "me@example.com"\npassword = "secret"\n',
+    )
+    got = extract_credentials([p])
+    assert got.username == "me@example.com"
+    assert got.password == "secret"
```

Ruff (0.15.9)

[error] Possible hardcoded password assigned to: "password"

(S105)